test/README.md

```diff
@@ -74,6 +74,8 @@ the same architecture, as the one intended for the generated test
 cases. In other words, you need to generate e.g test cases for `aarch64`
 images on an `aarch64` host.
 
+These tests are executed from the `osbuild` repository in GitHub CI.
```

Comment:
Don't you mean GitLab CI?

Reply:
I meant GitHub CI b/c that's the primary trigger for all of the osbuild jobs. Image tests themselves indeed execute in GitLab so not sure what wording to use here.

Reply:
Well then maybe let's say something like that the tests are triggered by GitHub action which then runs the tests in GitLab CI?

Reply:
Let's leave it at that and merge this, there is a clear link from Github to Gitlab so this makes sense.